app/views/assets/_batch_asset_selection.html.erb

```diff
@@ -0,0 +1,135 @@
+          Deselect all <%= "#{(text_for_resource items.first).downcase.pluralize}" %> </a></label>
+        <% items.each do |item| %>
+          <%= render :partial => "assets/asset_checkbox_row",
+                     :object => item,
+                     :locals => { :html_classes => "publishing_options",
+                                  :block_published => publishing } -%>
+        <% end %>
+      </div>
+    </div>
+  <% end %>
+  <br/>
+</div>
+
+<div id="sorted_by_isa" >
+  <% unless @assets_not_in_isa.empty? %>
+    <div id="items_not_in_isa" style="display: inline;">
+      <%= render :partial => 'assets/isa-tree-toggle', locals: { cb_parent_selector: "div#items_not_in_isa" } -%>
+      <h2 style="display: inline;">Items not in ISA</h2>
+      <div class="isa-tree" style="border-left:none">
+        <label><a class="selectChildren" id="select_all_items_not_in_isa"
+                  data-cb_parent_selector='div#items_not_in_isa' >
+          Select all items not in ISA </a></label> |
+        <label><a class="deselectChildren" id="deselect_all_items_not_in_isa"
+                  data-cb_parent_selector='div#items_not_in_isa' >
+          Deselect all items not in ISA </a></label>
+      </div>
+      <% @assets_not_in_isa.each do |item| %>
+        <%= render :partial => "assets/asset_checkbox_row",
+                   :object => item,
+                   :locals => { :html_classes => "publishing_options",
+                                :block_published => publishing } -%>
+      <% end %>
+      <br/>
+    </div>
+  <% end %>
+
+  <br/>
+
+  <% unless @investigations.empty? %>
+    <div id="items_in_isa" style="display: inline;">
+      <%= render :partial => 'assets/isa-tree-toggle', locals: { cb_parent_selector: "div#items_in_isa" } -%>
+      <h2 style="display: inline;">Items in ISA</h2>
+      <div class="isa-tree" style="border-left:none">
+        <label><a class="selectChildren" id="select_all_items_in_isa"
+                  data-cb_parent_selector='div#items_in_isa' >
+          Select all items in ISA </a></label>
+        | <label><a class="deselectChildren" id="deselect_all_items_in_isa"
+                  data-cb_parent_selector='div#items_in_isa' >
+          Deselect all items in ISA </a></label>
+        || <label><a class="collapseChildren" id="collapse_isa" data-cb_parent_selector='div#investigation_list' >
+          Collapse all </a></label>
+        | <label><a class="expandChildren" id="expand_isa" data-cb_parent_selector='div#items_in_isa' >
+          Expand all </a></label>
+        <% if show_hide_blocked %>
+          || <label><a class="showBlocked" data-cb_parent_selector='div#items_in_isa' data-blocked_selector="<%="#{blocked_selector}"%>">
+            Show blocked items </a></label>
+          | <label><a class="hideBlocked" data-cb_parent_selector='div#items_in_isa' data-blocked_selector="<%="#{blocked_selector}"%>">
+            Hide blocked items </a></label>
```

Comment:
These controls might be nicer be styled as buttons instead of plain links

Reply:
Addressed in [9b8185e](https://github.com/seek4science/seek/pull/1551/commits/9b8185eb473f4fdf47cdeb54742b017a3ba5d96e).